Content.Server/_Citadel/Shipyard/Commands/PurchaseShuttleCommand.cs

```diff
@@ -0,0 +1,44 @@
+using Content.Server.Administration;
+using Content.Server.Maps;
+using Content.Server.Shipyard.Systems;
+using Content.Shared.Administration;
+using Robust.Shared.Console;
+
+namespace Content.Server.Shipyard.Commands;
+
+/// <summary>
+/// purchases a shuttle.
+/// </summary>
+[AdminCommand(AdminFlags.Fun)]
+public sealed class PurchaseShuttleCommand : IConsoleCommand
+{
+    public string Command => "purchaseshuttle";
+    public string Description => "spawns and docks a specified shuttle from a grid file";
+    public string Help => $"{Command}";
```

Comment:
Should use a translation. Also needs to include the command's expected arguments.